src/i18n.ts

```diff
@@ -30,11 +32,10 @@ export const languages = [
 let currentLanguage = languages[0];
 let currentLanguageData = {};
 const fallbackLanguage = languages[0];
-const fallbackLanguageData = require("./locales/en.json");
 
 export const setLanguage = async (newLng: string | undefined) => {
   currentLanguage =
-    languages.find((language) => language.lng === newLng) || fallbackLanguage;
+    languages.find(language => language.lng === newLng) || fallbackLanguage;
```

Comment:
Wanted to raise a quick PR from Code Sandbox, but I might have to do a local clone to fix this up :/

Reply:
Yea, the CSB doesn't seem to work with our linter correctly :/